articles/sap/center-sap-solutions/deploy-s4hana.md

```diff
@@ -26,6 +26,12 @@ In this how-to guide, you'll learn how to deploy S/4HANA infrastructure in *Azur
 - A **User-assigned managed identity** which has Contributor role access on the Subscription or atleast all resource groups (Compute, Network,Storage). If you wish to install SAP Software through the Azure Center for SAP solutions, also provide Storage Blob data Reader, Reader and Data Access roles to the identity on SAP bits storage account where you would store the SAP Media.
 - A [network set up for your infrastructure deployment](prepare-network.md).
 - Availability of either Standard_D4ds_v4 or Standard_E4s_v3 SKUS which will be used for "Deployer VM".
+- - [Review the quotas for your Azure subscription](../../quotas/view-quotas.md). If the quotas are low, you might need to create a support request before creating your infrastructure deployment. Otherwise, you might experience deployment failures or an **Insufficient quota** error. 
```

Comment:
```suggestion
- [Review the quotas for your Azure subscription](../../quotas/view-quotas.md). If the quotas are low, you might need to create a support request before creating your infrastructure deployment. Otherwise, you might experience deployment failures or an **Insufficient quota** error. 
```